src/bin/am/commands.rs

```diff
@@ -40,6 +44,7 @@ pub async fn handle_command(app: Application, mp: MultiProgress) -> Result<()> {
     match app.command {
         SubCommands::Start(args) => start::handle_command(args, mp).await,
         SubCommands::System(args) => system::handle_command(args, mp).await,
+        SubCommands::Discord => Ok(open::that("https://discord.gg/kHtwcH8As9")?),
```

Comment:
Maybe instead of  returning a error we could just print the message: "Unable to open browser, open the following URL in your brower: {url}"